lib/puppet/type/foreman_smartproxy.rb

```diff
@@ -76,4 +76,13 @@ def refresh
     end
   end
 
+  autorequire(:class) do
+    ['Foreman::Service']
+  end
+
+  # puppetlabs-apache always defines the service httpd, even if the name is
+  # different.
+  autorequire(:service) do
+    ['httpd']
+  end
```

Comment:
I'm less familiar with autorequire -- does this continue to just work for proxies on servers without Apache?